src/middlewared/middlewared/plugins/apps/rollback.py

```diff
@@ -1,7 +1,10 @@
+import logging
```

Comment:
superfluous changes here. No reason to import logging and no reason to add a space between middlewared imports